src/schema/mondo.yaml

```diff
@@ -1,99 +1,360 @@
 id: https://w3id.org/mondo
 name: mondo
-description: >-
```

Comment:
@matentzn @sabrinatoro Updated `src/schema/mondo.yaml`. Please take a look.

I was able to prevent it from re-ordering things, but it still does some code style reformatting, sorry about that. Hope this is ok.

I scripted the edits starting with a map between the fields in the [GoogleSheet](https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180) and the `.yml:
```py
SHEET_YML_MAPPINGS = {
	'code_name': 'title',
	'definitions': 'description',
	'notes': 'notes',
	'examples': 'examples',
	'exclude_children': 'exclude_children',
}
```

@sabrinatoro If you want, we can streamline this and make the column names in the GoogleSheet the same as in the `.yml`. But I don't mind doing it this way.

If these mappings are incorrect, though, or if additional fields need to be added, please let me know. The script will ignore any values that these fields previously had in the `.yml`, and will completely overwrite with what is in the GoogleSheet, since it is the source of truth.

@sabrinatoro If you add any new columns to the GoogleSheet, it will add the exact spelling of the column name and use that as the field in the `.yml`.